### PR TITLE
Feature contact form: Added Contact form in Portfolio 

### PR DESCRIPTION
**Pull request by vikipretium**
### Added Contact form in Portfolio 

- Added contact form in the footer ✅ 
- Added  field accepts up to 30 chars ✅ 
- Added Email field accepts only emails ✅ 
- Message field accepts up to 500 chars ✅ 
- All fields have placeholder text ✅ 
- All fields have the correct type ✅ 
- Added required attribute to all inputs to verfiy no fields left empty ✅ 
- used Flexbox ✅ 
- No JavaScript ✅ 
- Submit Button Interaction ✅ 
-  Pair-programming commits ✅

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,313 +1,299 @@
...
+          <hr />
+        </li>
+      </ul>
+    </section>
+    <!-- Languages skills frameworks ends -->
+  </main>
+
+  <footer>
+    <div class="contact-form">
+      <h2 class="form-heading">Contact me</h2>
```
Comment (rotimiazeez) [inline index.html:265]: - [x] Kindly make sure all backgrounds designs are implemented to match the Figma design 👍☺️
Your Design | Figma Design
:---:|:---:
 |

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team🙋, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (vikipretium) [reply index.html:265]: Hello @rotimiazeez, 
The background image is loading in my local machine's browser. 
Can you help me to find whats the problem here?

Comment (vikipretium) [reply index.html:265]: My mistake @rotimiazeez, 
image
I just entered the file format two times 😅 .

**Review by Meltrust — APPROVED** [review]
Hi @vikipretium, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**